docs/user-guide/rules/no-insecure-cookies.md

```diff
@@ -0,0 +1,174 @@
+# Validate `Set-Cookie` Header (`validate-set-cookie-header`)
```

Comment:
Everywhere else the rule is called `no-insecure-cookies`. This rule has added more features than the insecure ones so I agree that `validate-set-cookie-header` is a better name but it should be changed everywhere (`.sonarrc`, `docs/user-guide/rules/index.md`, and rule file and tests).